tensorflow/core/api_def/base_api/api_def_Acos.pbtxt

```diff
@@ -5,5 +5,12 @@ op {
 Given a tensor `x`, this operation returns a tensor containing the 
 arccosine of each element in `x` or in other words it returns the 
 inverse of cos of the value.
+
+For Example :
+```python
```

Comment:
Please follow the indications at https://www.tensorflow.org/community/contribute/docs_ref and make this a testable docstring.

Please make sure there is a empty line before the doctest. There is no need for an empty line at the end of the doctest if it is the end of the proto.

Reply:
> Please make sure there is a empty line before the doctest. There is no need for an empty line at the end of the doctest if it is the end of the proto.

Is this all or should i remove the backticks (```) and put >>>

Reply:
You should use `>>>` instead of the backticks. Then the test is of the form

```
>>> python_code_goes_here
answer_goes_here
```

Reply:
@mihaimaruseac  can you once check if it is correct ?